samples/scenarios/jdbc_kerberos/README.md

```diff
@@ -0,0 +1,152 @@
+
+# Readme
+
+The postgres jdbc kerberos sample connector plugin demonstrates how to incorporate different kerberos-based authentication modes for your "Tableau + Postgres + JDBC" use cases. This sample plugin code has been tested for all the authentication methods listed here with all platforms (Mac Desktop, Windows Desktop and Server, Linux Server). However, the individual sections of the plugin components concern specific authentication modes and have been laid out here for the purposes of understanding and re-use. While publishing from Tableau Desktop, the Server RunAs and Viewer credentials authentication options will be visible in publish dialog only if you have logged into tableau using Integrated Authentication. 
+
+Contents
+1. [Use case(s) of this connector plugin](#purpose)
+1. [Introduction to Kerberos on Java](#intro) 
+1. [Kerberos SSO for Tableau Desktop](#desktop)
+1. [Kerberos RunAs and Delegation on Server](#server).
+
+
+## <a id="purpose"/> Use case(s) of this connector plugin
+1. Use **SSPI** authentication to connect a postgres database with `Tableau Desktop on Windows`   
+1. Use **Kerberos SSO via GSS-API** to connect a postgres database with `Tableau Desktop on Mac`.
+1. Publish a viz from `Tableau Desktop` using **Server RunAs authentication** and open it in Tableau Server. In this scenario, Tableau Server connects to databases with a service account.
+1. Publish a viz from `Tableau Desktop` using **Kerberos Delegation/Viewer Credentials authentication** and open it in Tableau Server. When a user accesses this viz on Tableau Server, the impersonation_runas_principal would be used to delegate queries to the tableau server user. 
+
+Below I give a brief introduction on how Java uses GSS-API to implement Kerberos authentication.
+
+## <a id="intro"> Introduction to Kerberos on Java
+
+### GSS-API
```

Comment:
I'm not sure if explaining the difference between sspi and gssapi is necessary now. It might just confuse them since Postgres JDBC has an option for sspi, but they should always use gssapi. 

You might just add a note that says to always use gssapi, and that on Windows Tableau uses sspi_bridge to translate from gssapi to sspi. And then they don't have to spend much time thinking about it.

Reply:
removed this section entirely. 
It looks cleaner now 
https://github.com/tableau/connector-plugin-sdk/tree/dsoni_postgres_kerb/samples/scenarios/jdbc_kerberos